articles/template-comparison.dd

```diff
@@ -206,7 +206,7 @@ template&lt;&gt; class factorial&lt;1&gt;
             based on Template Arguments)
             $(TD Yes:
 ---
-template void foo(T)(T i)
+void foo(T)(T i)
 {
   static if (can_fast_foo!(T))
```

Comment:
The C++ example is broken, btw: 

![1000003483](https://github.com/dlang/dlang.org/assets/3475150/ca7eba75-a426-4b45-9be8-434c9316b222)

Reply:
Nevermind, I later saw that you fixed that in here: https://github.com/dlang/dlang.org/pull/3688